### DISCUSSION TITLE
Пересчет таблицы по изменению в хедере

### DISCUSSION THREAD

**Discussion by konditerov** [totumonline/totum-ru-issues-and-discussions#567] [Как это сделать?] [answered]
Добрый день, 
Собираю таблицу с поставками товара, планирую сделать цикл с таблицами где от каждой даты и клиента формировать просчет поставки.
В хедере планировал указывать курс валюты для калькуляции поставки.
В силу удобства - удлобно не для каждой sku выводить курс, а для всего расчета.
Хотелось что бы при любом изменении данных в таблице, она синхронно пересчитывалась.

Читал в документации и на форуме, что это считается плохим тоном, но не нашел другого решения. Таблица получается не большой для частого пересчета.

Подскажите в какую сторону смотреть в таком случае?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Те внутри цикла вы в хедере указываете курсы и дальше для товаров в этом цикле рассчитываете стоимости. Если так то наоборот - это самая удобная реализация.

**Reply by konditerov** [reply]
> Те внутри цикла вы в хедере указываете курсы и дальше для товаров в этом цикле рассчитываете стоимости. Если так то наоборот - это самая удобная реализация.

Да, но, при изменении хедера - пересчет столбца не происходит, хотелось оживить. Сейчас я в обычной таблице в таблице цикла пробую сделать
При изменении данных в строке - пересчитывается.

**Reply by ttm-alexeypolunin** [reply]
Если это внутри цикла то там тип таблицы Рассчетная - она пересчитывается целиком всегда.
